renderdoc/driver/d3d12/d3d12_common.h

```diff
@@ -726,6 +726,7 @@ enum class D3D12Chunk : uint32_t
   Device_CreateSampler,
   Device_CopyDescriptors,
   Device_CopyDescriptorsSimple,
+  Device_OpenSharedHandle,
```

Comment:
Adding the chunk into the middle of the list like this will break opening old captures, as the serialised chunk IDs won't match. Any new chunks should be added to the end of the list.

Reply:
I added a compile time assert in d3d12_stringise.cpp that checks the last added chunk type.